noether_gui/ui/tpp_pipeline_widget.ui

```diff
@@ -20,10 +20,25 @@
    <string>Form</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
+   <property name="spacing">
+    <number>0</number>
+   </property>
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
+    <number>0</number>
+   </property>
```

Comment:
It sets lowers the space between the QWidget and the layout components inside from 7px to 0px, so that the pipelines settings have more space in the screen.